### PR TITLE
Create setup for the project

### PR DESCRIPTION
**Pull request by jmnyarega**
# What the PR does
 - project setup

### CODE REVIEW

File: webpack.config.dev.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```
Comment (kimani-ndegwa) [inline webpack.config.dev.js]: Where is the config that determines the different environment constraints for picking a webpack file?

File: webpack.config.prod.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      },
+    ],
+  },
+  plugins: [
+    new webpack.NoEmitOnErrorsPlugin(),
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV || 'production'),
```
Comment (jmnyarega) [inline webpack.config.prod.js]: I manually passed  types of environments on each of the config files

Comment (jmnyarega) [reply webpack.config.prod.js]: So whenever I ran `yarn build`  this script is run `webpack -p --config=webpack.config.prod.js`, this script references configurations for the production environment

If i ran `yarn start` this script is run `webpack-dev-server --config=webpack.config.dev.js --open --history-api-fallback` this script references configurations for the development environment.

The scripts are defined inside the package.json file.

@kimani-ndegwa

Comment (jmnyarega) [reply webpack.config.prod.js]: Great.

**Pull request merged**